packages/orbit-components/src/Tabs/TabContext.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from "react";
```

Comment:
I would rather have the apps control the state of the tabs individually. I think having a Context and trying to control the tabs behavior not only is more expensive but is also less flexible. It would also remove the need of iterating over children.
The basic Tabs usage is rather simple to control on the app side, and it simplifies the component's logic and implementation by a lot. I honestly don't think having a Context is necessary here. I would let the state management to the user side.